### DISCUSSION TITLE
Mini display port is not working on mac os monterey

### DISCUSSION THREAD

**Discussion by zJairO** [MultimediaLucario/Lenovo-ThinkPad-T440s#32] [Q&A]
Hey there, I was using Mac os big sur and had no problems with mini display port, could not get 4k. 

But now I upgraded to mac os Monterey, everything is going better than before, just my mini display port to display port adapter is no longer working, when I connect it my monitor turns on but after it, it shutdowns.

Do you know how to fix this? Thank you.

![Screen Shot 2023-04-08 at 23 35 54](https://user-images.githubusercontent.com/5152239/230756303-0226b383-8459-400a-bbc0-cda78f5287f2.png)

**Comment by MultimediaLucario** [comment]
What version of OpenCore are you running?

**Reply by MultimediaLucario** [reply]
It works the best on Version 0.8.8.

**Comment by MultimediaLucario** [comment]
Also what is you CPU configuration (ie i5-4200u, i5-4210u, i5-4300u,etc.)?

**Comment by MultimediaLucario** [comment]
Also, did you disable Secure Boot?

**Comment by MultimediaLucario** [comment]
I ask this because those could be some factors in the issue.

**Comment by zJairO** [comment]
Good morning.

Procesor is Intel Core i5-4300U

Secure boot is disabled, I can not activate it.

Going to check if opencore 0.8.8 works, I had 0.9.0..

Thanks.
